app/models/profile.rb

```diff
@@ -57,6 +58,11 @@ class Profile < ApplicationRecord
     open_to_work: 2
   }
 
+  enum visibility: {
+    private: 0,
+    public: 1
+  }, _suffix: true
```

Comment:
`suffix` is necessary because `public` and `private` are keywords. So this means the methods are written like `profile.private_visibility?`